test/streaming/get/pb_creators.py

```diff
@@ -356,6 +363,50 @@ def town_detail():
     return builds(PBTownDetail, name=text(alphabet=ALPHANUM, max_size=TEXT_MAX_SIZE), stateOrProvince=text(alphabet=ALPHANUM, max_size=TEXT_MAX_SIZE))
 
 
+#####################################
+# IEC61968 infIEC61968 InfAssetInfo #
+#####################################
+
+def current_transformer_info():
```

Comment:
suggestion: although it is not needed, to be consistent with the changes you made to cim_creators, you could add all of these to an __all__ collection

Reply:
`__all__` added. Constants e.g. `MIN_32_BIT_INTEGER` are excluded just as in `cim_creators.py`. They may be refactored out in the future as they are identical between the two.